gensrc/proto/binlog.proto

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
```

Comment:
just use proto3?

Reply:
`PageHeaderPB` uses an enum `CompressionTypePB` defined with proto2.  As [proto3#using_proto2_message_types](https://developers.google.com/protocol-buffers/docs/proto3) says, `proto2 enums cannot be used directly in proto3 syntax`, so keep using proto2 here